certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+                    "value-title": "name"
+                  }
+                },
+                "stage_id": {
+                  "type": "integer",
+                  "description": "",
+                  "title": "Approval Stage",
+                  "format": "string",
+                  "x-ms-dynamic-list": {
+                    "operationId": "GetProcessApprovalStages",
+                    "itemValuePath": "id",
+                    "itemTitlePath": "name",
+                    "parameters": {
+                      "process_id": {
+                        "parameterReference": "body/process_id"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetProcessApprovalStages",
+                    "value-path": "id",
+                    "value-collection": "msg",
+                    "value-title": "name",
+                    "parameters": {
+                      "process_id": {
+                        "parameter": "process_id"
+                      }
+                    }
+                  }
+                }
+              },
+              "required": [
+                "target_url",
+                "operation",
+                "process_id",
+                "stage_id"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/subscribe_trigger2": {
+      "x-ms-notification-content": {
+        "description": "Trigger Payload",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "ticket_id": {
+              "type": "string",
+              "description": "ticket ID"
+            },
+            "process_name": {
+              "type": "string",
+              "description": "Process Name"
+            },
+            "stage_name": {
+              "type": "string",
+              "description": "Stage Name"
+            },
+            "sla_metrix": {
+              "type": "string",
+              "description": "SLA Metrix"
+            },
+            "sla_breach_duration": {
+              "type": "string",
+              "description": "SLA breach duration"
+            },
+            "date_created": {
+              "type": "string",
+              "description": "Date Created"
+            },
+            "date_breached": {
+              "type": "string",
+              "description": "Date Breached"
+            },
+            "view_link": {
+              "type": "string",
+              "description": "Link of view"
+            },
+            "stage_approvers": {
+              "type": "string",
+              "description": "Users that can action"
+            }
+          }
+        },
+        "headers": {
+          "Content-Type": {
+            "description": "Content-Type",
+            "type": "string"
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.